### ISSUE TITLE
FURB157 misses `decimal.Decimal("_-1")`

### ISSUE THREAD

**Issue by dscorbett** [astral-sh/ruff#21186]
### Summary

[`verbose-decimal-constructor` (FURB157)](https://docs.astral.sh/ruff/rules/verbose-decimal-constructor/) has a false negative when the sign character follows an underscore. [Example](https://play.ruff.rs/34abd766-bcd1-4e22-9a21-aa2d340f859f):
```console
$ cat >furb157.py <<'# EOF'
from decimal import Decimal
Decimal("_-1")
# EOF

$ ruff --isolated check furb157.py --select FURB157
All checks passed!
```
This was a true positive in version 0.14.2.

### Version

ruff 0.14.3 (8737a2d5f 2025-10-30)
